aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -1182,13 +1182,6 @@ module aptos_token::token {
 
         // we add more tokens with property_version 0
         let token_id = create_token_id(token_data_id, 0);
-        deposit_token(account,
-            Token {
-                id: token_id,
-                amount,
-                token_properties: property_map::empty(), // same as default properties no need to store
-            }
-        );
         event::emit_event<MintTokenEvent>(
```

Comment:
Is this backwards compatible? If `deposit_token` aborted in the new version, a `MintTokenEvent` will be emitted, whereas in the previous version it would not have been emitted.

Reply:
these events are all in a txn, which is atomic. Either all make it or none make it.